Problem with Element_wise_2 Div

Trying to convert a modified bert tiny model and run on host. `model.c` file gets generated with no errors, but when compiling with gcc, compiler complains about division in element wise division operator.
```
> gcc -O3 -std=c99 main.c -lm -o bert_tiny_new_simplified_run                                                                         
In file included from main.c:8:
bert_tiny_new_simplified_model.c: In function ‘node__bert_encoder_layer_0_attention_self_Div’:                                                                                   
bert_tiny_new_simplified_model.c:34122:53: error: expected ‘;’ before ‘static’
34122 |                 C[i0][i1][i2][i3] = A[0][i1][i2][i3]/*B;;
      |                                                     ^
      |                                                     ;
main.c:44:1: error: expected declaration or statement at end of input
   44 | }
      | ^
```
The part of the `model.c` file it's complaining about is below:
```
/*
 * Operand:           Div
 * Name in ONNX file: /bert/encoder/layer.0/attention/self/Div
 */
FUNC_PREFIX void node__bert_encoder_layer_0_attention_self_Div( const float A[1][2][128][128], const float *B, float C[1][2][128][128] )
{
        /* Div
           Implemented with Elementwise_2 template.
           Attributes (these are the union of attributes for all 2-element-wise
                       operands. So most likely these values are ignored by onnx2c).
           shift_dir: NOT_GIVEN
           fmod: 0
         */
        for (unsigned i0=0; i0<1; i0++) {
        for (unsigned i1=0; i1<2; i1++) {
        for (unsigned i2=0; i2<128; i2++) {
        for (unsigned i3=0; i3<128; i3++) {
                C[i0][i1][i2][i3] = A[0][i1][i2][i3]/*B;;
        }
        }
        }
        }
}

/*
 * Operand:           Add
```
Judging by the colors on my file editor, and the compiler message, I suspect the compiler thinks that the division followed by the de-referencing operator is the start of a comment block. 
I did some looking and from the implementation of the "Div" Elementwise_2, it seems to me that `*B` is being passed as the `b` argument, and thus being placed right next to the forward slash.
```
else if (op == "Div" )
			operation = [](const std::string& a, const std::string& b)
				{ return  a+"/"+b+";"; };
``` 
For a manual fix, adding a space after the slash gets rid of the issue. Will post a pull request with a fix for elementwise_2.h

--- a/src/nodes/elementwise_2.h
+++ b/src/nodes/elementwise_2.h
@@ -39,7 +39,7 @@ class Elementwise_2 : public Node {
 				};
 		else if (op == "Div" )
 			operation = [](const std::string& a, const std::string& b)
-				{ return  a+"/"+b+";"; };
+				{ return  a+"/ "+b+";"; };
 		else if (op == "Equal" ) {
 			output_is_bool = true;
 			// NB: specs don't define what kind of equality is meant when inputs are floating point

